:sparkles: #2010: Modified images and text according to new Code dropdown changes (#2013)
platformsh/platformsh-docs: docs/src/integrations/source/bitbucket.md

## Clones and commits

> You can clone your codebase by going to Console > Code > GIT and using the git clone command, or you can run `platform get <projectID>` with the CLI.

Reviewer: I'm not sure that the change to a how to was completely necessary. I think it worked as a reference, it just needed a change to the drop-down. Haven't looked at the articles in depth though. :shrug: 

If sticking with this way here and below, I'd suggest a slightly different wording:

```suggestion
You can clone your codebase by running `platform get <projectID>` or in your project in the console going to Code > Git and running the `git clone` command.
```

The CLI way seems easier and in keeping with the rest of the article, though you can make that second if you strongly prefer.

But given that "Console" isn't something that appears in the UI like the other items in the list, I wouldn't have them next to each other like that.

When you perform this action, you are actually cloning from your remote integrated repository,
so long as you have the [appropriate access to do so](/administration/users.md#user-access-and-integrations).

Your Bitbucket repository is considered by Platform.sh to be the "source of truth" for the project.

> The project is only a mirror of that repository, and all commits should be pushed only to Bitbucket.

Reviewer: ```suggestion
The project is only a mirror of that repository and all commits should be pushed only to Bitbucket.

```

Missing empty line at the end of the file :wink:
